apps/209-datepicker/app.R

```diff
@@ -0,0 +1,158 @@
+### Keep this line to manually test this shiny application. Do not edit this line; shinycoreci::::is_manual_app
+
+library(shiny)
+library(magrittr)
+library(rlang)
+
+options(shiny.minified = FALSE)
```

Comment:
Do you want to reset this option `onStop()`?

Reply:
Oh, this line actually isn't needed.